packages/react/package.json

```diff
@@ -64,13 +64,17 @@
     "classnames": "2.3.1",
     "deepmerge": "4.2.2",
     "lodash": "4.17.21",
+    "mapbox-gl": "npm:empty-npm-package@1.0.0",
```

Comment:
We need to alias `mapbox-gl` for users, even though we overwrite it with `maplibre-gl`. `react-map-gl` recommends aliasing the package with an empty npm package: https://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork